packages/dashboard/src/components/iot-dashboard/iot-dashboard-internal.tsx

```diff
@@ -125,12 +127,19 @@ export class IotDashboardInternal {
       widgets: this.dashboardConfiguration.widgets.filter(({ id }) => this.selectedWidgetIds.includes(id)),
     });
   };
+
   onSendToBack = () => {
     this.sendToBack({
       widgets: this.dashboardConfiguration.widgets.filter(({ id }) => this.selectedWidgetIds.includes(id)),
     });
   };
 
+  onClearSelection = () => {
+    this.selectWidgets({
```

Comment:
Are we planning to have integration tests for this type of functionality?

Reply:
ahh good catch, I missed those tests. will add.